docs/syscall_descriptions_syntax.md

```diff
@@ -95,7 +96,8 @@ example_struct {
 	f0	int8			# random 1-byte integer
 	f1	const[0x42, int16be]	# const 2-byte integer with value 0x4200 (big-endian 0x42)
 	f2	int32[0:100]		# random 4-byte integer with values from 0 to 100 inclusive
-	f3	int64:20		# random 20-bit bitfield
+	f3	int32[1:10, 2]		# random 4-byte integer with values {1, 3, 5, 7, 9}
```

Comment:
@dvyukov Have you seen that example?
I'm wondering if we're on the same page regarding the semantics of the alignment. I've implemented it to be an alignment starting from `beginRange`, not from 0. Your other comments seem to assume its an alignment to 0.
Aligning on `beginRange` seemed more generic; for instance, it allows us to express odd values as above. It may need to be said explicitly in the description though.

Reply:
I agree. It should be alignment from range begin.